src/backend/aspen/app/views/phylo_runs.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import json
+import os
+import re
+from typing import Iterable, MutableSequence
+
+import sentry_sdk
+from flask import g, jsonify, request, Response
+from marshmallow import fields, Schema, validate
+from marshmallow.exceptions import ValidationError
+from sqlalchemy.orm import joinedload
+
+from aspen import aws
+from aspen.app.app import application, requires_auth
+from aspen.app.views.api_utils import authz_sample_filters
+from aspen.database.models import (
+    AlignedGisaidDump,
+    PathogenGenome,
+    PhyloRun,
+    Sample,
+    Workflow,
+    WorkflowStatusType,
+)
+
+# What kinds of nextstrain build configs does this endpoint support?
+PHYLO_TREE_TYPES = {
+    "local": "local.yaml",
+    "contextual": "contextual.yaml",
+}
+
+
+class PhyloRunRequestSchema(Schema):
+    name = fields.String(required=True, validate=validate.Length(min=1, max=128))
+    samples = fields.List(fields.String(), required=True)
+    tree_type = fields.String(
+        required=True, validate=validate.OneOf(PHYLO_TREE_TYPES.keys())
+    )
+
+
+class GroupResponseSchema(Schema):
+    id = fields.Int()
+    name = fields.String()
+    address = fields.String()
+    prefix = fields.String()
+    division = fields.String()
+    location = fields.String()
+
+
+class WorkflowStatusSchema(Schema):
+    name = fields.String()
+
+
+class PhyloRunResponseSchema(Schema):
+    id = fields.Int()
+    start_datetime = fields.DateTime()
+    end_datetime = fields.DateTime()
+    workflow_status = fields.Pluck(WorkflowStatusSchema, "name")
+    group = fields.Nested(GroupResponseSchema, only=("id", "name"))
+    template_file_path = fields.String()
+    template_args = fields.Nested(GroupResponseSchema, only=("division", "location"))
```

Comment:
this is a huge nit, but if we're starting to use serializers like this it might be nice to separate them out somewhere away from our views (like on the same level as models/views we could also have serializers)